Original patch from: ../4.3.2/140-netbsd-symbolic.patch

-= BEGIN original header =-
Original patch from gentoo: gentoo/src/patchsets/gcc/4.3.1/gentoo/11_all_gcc-netbsd-symbolic.patch
http://bugs.gentoo.org/122698

-= END original header =-

diff -durN gcc-4.3.3.orig/gcc/config/netbsd-elf.h gcc-4.3.3/gcc/config/netbsd-elf.h
--- gcc-4.3.3.orig/gcc/config/netbsd-elf.h	2007-09-03 18:14:04.000000000 +0200
+++ gcc-4.3.3/gcc/config/netbsd-elf.h	2009-01-27 22:19:18.000000000 +0100
@@ -82,6 +82,7 @@
 #define NETBSD_LINK_SPEC_ELF \
   "%{assert*} %{R*} %{rpath*} \
    %{shared:-shared} \
+   %{symbolic:-Bsymbolic} \
    %{!shared: \
      -dc -dp \
      %{!nostdlib: \
